.github/workflows/test-android-device.yaml

```diff
@@ -5,14 +5,17 @@ on:
   schedule:
     - cron: '30 21 * * *'
 
+env:
+  WORKFLOW_FRIENDLY_NAME: Test status (Android device on Firebase Test Lab)
```

Comment:
There's already a name, here:

```yaml
jobs:
  run_tests:
    name: Flutter ${{ matrix.flutter-version }} on Firebase Test Lab
```

Let's remove this env.
Pass `jobs.run_tests.name` to `send-slack-message`'s `WORKFLOW_FRIENDLY_NAME` instead.